src/patternfly/components/Alert/alert-icon.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="pf-c-alert__icon{{#if alert-icon--modifier}} {{alert-icon--modifier}}{{/if}}"
+  {{#if alert-icon--attribute}}
+    {{{alert-icon--attribute}}}
+  {{/if}}>
+  {{> @partial-block}}
+  {{#if pf-c-alert-icon--success}}
```

Comment:
can you remove the `pf-c` from the icons too

Reply:
done